.github/workflows/lint-and-tests.yml

```diff
@@ -127,13 +126,7 @@ jobs:
             github.event.pull_request.head.ref != 'chore/crowdin')
         # We want to enforce that the actual `turbo@latest` package is used instead of a possible hijack from the user
         # the `${{ needs.base.outputs.turbo_args }}` is a string substitution happening from the base job
-        run: npx --package=turbo@latest -- turbo lint ${{ needs.base.outputs.turbo_args }}
-
-      - name: Run `turbo prettier`
-        if: steps.eslint-step.outcome == 'success'
-        # We want to enforce that the actual `turbo@latest` package is used instead of a possible hijack from the user
-        # the `${{ needs.base.outputs.turbo_args }}` is a string substitution happening from the base job
-        run: npx --package=turbo@latest -- turbo prettier ${{ needs.base.outputs.turbo_args }}
+        run: npx --package=turbo@latest -- turbo lint:js lint:md lint:css prettier ${{ needs.base.outputs.turbo_args }
```

Comment:
You forgot the second closing brace in `${{ needs.base.outputs.turbo_args } //there needs to be another one here`.

Everything else looks good!

Reply:
@anthonyshew can you fix that one? 👀

Reply:
Oops! Thanks for that catch, @rikhall1515!